src/common/system.cpp

```diff
@@ -113,10 +113,15 @@ int GetNumCores()
 
 std::optional<size_t> GetTotalRAM()
 {
-    auto clamp{[](uint64_t v) { return size_t(std::min(v, uint64_t{std::numeric_limits<size_t>::max()})); }};
+    [[maybe_unused]] auto clamp{[](uint64_t v) { return size_t(std::min(v, uint64_t{std::numeric_limits<size_t>::max()})); }};
 #ifdef WIN32
     if (MEMORYSTATUSEX m{}; (m.dwLength = sizeof(m), GlobalMemoryStatusEx(&m))) return clamp(m.ullTotalPhys);
-#elif defined(__linux__) || defined(__APPLE__)
+#elif defined(__APPLE__) || \
+      defined(__FreeBSD__) || \
+      defined(__NetBSD__) || \
+      defined(__OpenBSD__) || \
+      defined(__illumos__) || \
+      defined(__linux__)
```

Comment:
4ebf456dc589a993faa73b8b377c0d1919fbd577

Could you please update the commit message so it reflects the changes?

Reply:
Done.